packages/headless/src/controllers/commerce/field-suggestions/headless-field-suggestions-generator.ssr.ts

```diff
@@ -17,7 +17,7 @@ export type {GeneratedFieldSuggestionsControllers} from './headless-field-sugges
 export type {FieldSuggestionsGenerator};
 
 export interface FieldSuggestionsGeneratorDefinition
-  extends SharedControllerDefinitionWithoutProps<FieldSuggestionsGenerator> {}
+  extends SearchAndListingControllerDefinitionWithoutProps<FieldSuggestionsGenerator> {}
```

Comment:
Should this controller, recent queries, and instant products be universal, too? They can be used in a standalone search box, which implies there doesn't necessarily need to be a listing or search interface on the current page.

In fact, there might not even be a recommendations interface on the page. It could be just a product description page whose only Coveo-driven component is a standalone search box.